### ISSUE TITLE
Not enough bandwidth

### ISSUE THREAD

**Issue by rvillalba-novetta** [Tedyst/HikLoad#4]
Hello, I've tried getting something similar to work and I keep getting an error from ffmpeg:

[rtsp @ 000001b3b14db040] method DESCRIBE failed: 453 Not Enough Bandwidth

This was also the case with this library. Any chance you've come across this before?

**Comment by Tedyst** [comment]
I did not have this problem before, but the response seems an error from the DVR/NVR. 453 from the RTSP Specification means 
```
The request was refused because there was insufficient bandwidth.
   This may, for example, be the result of a resource reservation
   failure.
```
What model of the device are you using? May be a config option in it.

**Comment by rvillalba-novetta** [comment]
Yeah that's the same conclusion I've reached. I've tried looking for settings/information regarding the NVR and streaming bandwidth, but to no avail thus far. I'm guessing perhaps there is something wrong with their implementation as they don't seem to use RTSP from any of their clients (web or native) when playing back tracks. The tracks will play in VLS every once in a while, but its rare. The live feeds will play through RTSP just fine. I've reached out to the manufacturer to see if they can give any additional info. 

Thank you very much for your quick response, I was just curious if you had come across this issue before. If I do end up finding out a solution I will follow up here with relevant info. Thanks again!
